### PR TITLE
[oneDNN]: Adding support for bf16 output in MklDequantize op

### PR DESCRIPTION
**Pull request by bhavani-subramanian**
This is needed for supporting int8-bf16 mixed precision models.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

File: tensorflow/core/ops/mkl_array_ops.cc [hunk 1]
```diff
@@ -102,7 +102,7 @@ REGISTER_OP("_MklDequantize")
     .Input("input: T")
     .Input("min_range: float")
     .Input("max_range: float")
-    .Output("output: float")
+    .Output("output: dtype")
```
Comment (cantonios) [inline tensorflow/core/ops/mkl_array_ops.cc:105]: Can you add a default of float?  Otherwise there will be a compatibility break.

**Review by cantonios — CHANGES_REQUESTED** [review]

File: tensorflow/core/ops/mkl_array_ops.cc (hunk 1 above)
Comment (bhavani-subramanian) [reply tensorflow/core/ops/mkl_array_ops.cc:105]: @cantonios I think it is already set to a default of float [here](https://github.com/tensorflow/tensorflow/pull/58020/files#diff-0e82ae77cc34257ccaf4ecca3a9514132fc211358058f09973cc082358f1b43cL110).

Comment (cantonios) [reply tensorflow/core/ops/mkl_array_ops.cc:105]: Ah, you're right, sorry.

**Review by cantonios — APPROVED** [review]

**Pull request merged**